Steps.txt

```diff
@@ -0,0 +1,54 @@
+Test Driven Extensions
+
+Current Sensing at high fidelity
+
+ - The current sensor has a 12-bit A2D converter.
+ - The sensor is capable of sensing a maximum temperature of 10 Amps.
+ - A reading of 0 means 0A and a reading of 4094 means 10 Amps.
+ - The readings in-between are linearly scaled.
+ - A reading of 4095 indicates an error.
+
+ - Your program needs to be compatible with 12-bit input. 
+   Assume that the 12-bit numbers are transmitted as an array of integers 
+   (no bit-level packing).
+
+responsibility of the extension
+
+Test driven development
+
+- Ignore the error readings
```

Comment:
Suggestion: In test-driven development, try wording the test as inputs and outputs.
That way things become specific. What if all readings are errors? Does 'ignore' mean 'pretend as if nothing happened'?